docs/development/testguide.rst

```diff
@@ -895,3 +895,25 @@ Python `float` objects and compared numerically.  This means that small
 differences in representation of roundoff digits will be ignored by the
 doctest.  The values are otherwise compared exactly, so more significant
 (albeit possibly small) differences will still be caught by these tests.
+
+
+Continuous integration
+----------------------
+
+Astropy uses `Travis <https://travis-ci.org/astropy/astropy>`_ for continuous
+integration on Linux and OSX setups, and `Appveyor
```

Comment:
"continous integration" -> "continuous integration (CI)" (because you use the acronym below, and some novices might not make that connection.